server/tests/integration/testPlanRun.test.js

```diff
@@ -15,17 +15,21 @@ let assertionResultId1;
 let assertionResultId2;
 let assertionResultId3;
 
-const getTestPlanVersionId = async () => {
+const getTestPlanVersionId = async (testFormatVersion = 1) => {
```

Comment:
A unit test for a v2 test plan version would be great. I assume it is going to happen with other work but just noting that desire here since I see a pathway opened up.

Reply:
I'll be adding v2 tests in a separate PR